### DISCUSSION TITLE
fatalDeprecations and verbose true not working on sass-loader

### DISCUSSION THREAD

**Discussion by raffi-46** [webpack/webpack#18759] [Q&A]
my version : 
![image](https://github.com/user-attachments/assets/c156902b-fc34-4bed-bf69-92d0fdcbf715)

![image](https://github.com/user-attachments/assets/2502a461-31e9-4693-8694-0fab0458e0c7)

Expected behaviour : fatalDeprecations must throw error on very deprecations as sass mentioned and verbose flag true will show all the deprecations instead of 5

Current behaviour : shows only 5 deprecations and fatalDeprecations not shows error 

IMP! : Or Is there any way to found all the list or error to fix or stop the build to fix and fix all until finished. Becuase I cant able to re build everytime to find this issuem as I have 2000+ deprecations to fix https://sass-lang.com/documentation/breaking-changes/mixed-decls/

@alexander-akait

**Comment by alexander-akait** [comment]
`fatalDeprecations` works only in modern api, to fix it you can 
- https://github.com/webpack-contrib/sass-loader?tab=readme-ov-file#api set to `modern`
- or update https://github.com/webpack-contrib/sass-loader/releases/tag/v16.0.0
